backend/leaderboard/models.py

```
@@ -510,6 +546,41 @@ def update_user_leaderboard_entries(user):
         LeaderboardEntry.update_leaderboard_ranks(leaderboard_type)
 
 
+@receiver(post_delete, sender=Contribution)
+def update_leaderboard_on_contribution_delete(sender, instance, **kwargs):
```

Ruff (0.15.20)

[warning] Unused function argument: `sender`

(ARG001)